pkg/render/common/elasticsearch/service.go

```diff
@@ -49,5 +50,12 @@ func HTTPSEndpoint(osType rmeta.OSType, clusterDomain string) string {
 }
 
 func ElasticEndpoint() string {
+	if ep := os.Getenv("ELASTIC_ENDPOINT"); ep != "" {
```

Comment:
This is making me rethink putting the elastic URL on the Tenant resource. We probably need a per-cluster fallback anyway, which sort of invalidates that concept. 

Unless I can think of a way to make this work, we'll probably need to switch back to a per-cluster ES URL.

Reply:
Should we add a task in the backlog ?